index.bs

```diff
@@ -332,13 +332,46 @@ The agent-info-response message contains the following properties:
     the device.  This is used mainly for debugging purposes, but may be
     displayed to the user of the requesting agent.
 
-: receives-audio (optional)
-:: The agent has to indicate that it supports audio. If false
-    or not included, it is assumed audio content is not supported.
-
-: receives-video (optional)
-:: The agent has to indicate that it supports video . If false
-    or not included, it is assumed video content is not supported.
+: capabilities (required)
+:: The contorol protocols, roles, and media types the agent support.
+    Presence indicates capability and absence indicates lack of
+    capability.  Capabilities should should affect how an agent is
+    presented to a user, such as drawing a different icon depending on
+    the media types it supports.
+
+The various capabilities have the following meanings:
+
+: receive-audio
+:: The agent can generally receive audio for the control protocols it
+    supports.  Each control protocol can have more specific capability
+    mechanisms, such as support for specific URLs in the presentation
```

Comment:
I'm not sure this is the best example since supporting the rendering of a URL is somewhat orthogonal to the ability to receive or play audio.  You could present a URL that plays no audio, for example.